ddtrace/tracer/example_test.go

```diff
@@ -25,27 +25,28 @@ func Example() {
 	defer tracer.Stop()
 
 	// Start a root span.
-	span, ctx := tracer.StartSpanFromContext(ctx, "your.work")
+	span, ctx := tracer.StartSpanFromContext(ctx, "parent")
 	defer span.Finish()
 
-	// Create a child, using the context of the parent span.
-	child, ctx := tracer.StartSpanFromContext(ctx, "this.service")
-	child.SetTag(ext.ResourceName, "alarm")
-
 	// Run some code.
-	err := yourCode(ctx)
-
-	// Finish the span.
-	child.Finish(tracer.WithError(err))
+	doSomething(ctx)
 }
 
-func yourCode(ctx context.Context) error {
+func doSomething(ctx context.Context) {
+	// Create a child, using the context of the parent span.
+	opts := []tracer.StartSpanOption{
+		tracer.Tag(ext.ResourceName, "alarm"),
+	}
```

Comment:
We don't need it, persay. It just seems pretty common that people have a lot of options that they'll want, so I thought it might be nice in the example. If you think it sets a bad example though, I'm happy to remove it. I don't feel strongly, so LMK what you think.

Reply:
I think it's fine to have a slice when needed, but no point to have it when _not_ needed. I'd keep it as simple as possible.